content/docs/reference/types/object.mdx

```diff
@@ -35,6 +39,22 @@ type ObjectField = {
 }
 ```
 
+## Usage
+
+Ultimately there are 3 ways to configure this type...
+
+* With `list` set to `false`, the `fields` array can be used to wrap some data – this appears in the editor as a nested page. This can be used to organise the CMS, such as grouping options together, as in the gif below.
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331124/tina-io/docs/Sep-26-2024_16-05-37_d2ol8w.gif)
+
+* With `list` set to `true`, the `fields` array contains the data structure to be repeated – this appears in the editor as above, but grouped as a list.
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331115/tina-io/docs/Sep-26-2024_16-03-00_f3ogev.gif)
+
+* With `list` set to `true`, and using `templates` as opposed to `fields` the user can select between associated templates – this appears as above, but pressing the `+`  gives you the option to choose between templates.
```

Comment:
can we use templates and fields alongside one another?

Reply:
Nah it's only one or the other

Reply:
✅ apoc it is not possible